sovryn-token-bridge/bridge/test/AllowTokens_test.js

```diff
@@ -360,8 +401,23 @@ contract('AllowTokens', async function (accounts) {
             assert.equal(result, true);
         });
 
+        it('should check if amount is smaller or equal to maxPerToken only if maxPerToken is set', async function() {
+            let newMax = web3.utils.toWei('2');
+            
+            await this.allowTokens.setFeeAndMinPerToken(this.token.address, web3.utils.toWei('0.5'), web3.utils.toWei('0.5'),{ from: manager }) ;
+            result = await this.allowTokens.isValidTokenTransfer(this.token.address, web3.utils.toWei('3'), 0, true);
+            assert.equal(result, true);
+
+            await this.allowTokens.setMaxPerToken(this.token.address, newMax ,{ from: manager });
+            result = await this.allowTokens.isValidTokenTransfer(this.token.address, web3.utils.toWei('3'), 0, true);
+            assert.equal(result, false);
 
+            result = await this.allowTokens.isValidTokenTransfer(this.token.address, web3.utils.toWei('1') , 0, true);
+            assert.equal(result, true);
 
+            result = await this.allowTokens.isValidTokenTransfer(this.token.address, web3.utils.toWei('2'), 0, true);
+            assert.equal(result, true);
```

Comment:
Maybe add one test here, `setMaxPerToken(this.token.address, 0, {from: manage})` and then check that the maximum is unset.